torch/_ops.py

```diff
@@ -32,28 +32,28 @@ def __init__(self, overloadpacket, op, schema):
         self._op = op
         self._schema = schema
         self._overloadpacket = overloadpacket
+        self.__name__ = 'default' if schema.overload_name is '' else schema.overload_name
 
     # it's a no-op since OpOverload object is immutable and must be unique for a given op overload.
     def __deepcopy__(self, memo=None):
         return self
 
     def __str__(self):
-        return "OpOverload(op='{}.{}', overload='{}')".format(*self._schema.name.split("::"), self.overload_name)
+        return "OpOverload(op='{}.{}', overload='{}')".format(*self._schema.name.split("::"), self.__name__)
 
     def __call__(self, *args, **kwargs):
         return self._op(*args, **kwargs or {})
 
     def __getattr__(self, key):
         return getattr(self._op, key)
 
-    # `my_namespace::my_op`
-    @property
-    def name(self):
-        return "{}.{}".format(*self._schema.name.split("::"))
+    def __hash__(self):
+      return hash(self._op)
 
+    # `my_namespace.my_op_name.overload_name`
     @property
-    def overload_name(self):
-        return self._schema.overload_name
+    def name(self):
```

Comment:
does someone have a name suggestion for this?

Reply:
I actually would suggest this as `__str__` and your current `__str__` become `__repr__`

Reply:
but usually repr should print something that can be used to construct the class object right? This is not true for the current `__str__` return

Reply:
This property is currently called `qualified_op_name` for the `OpOverloadPacket` btw.

Reply:
ok so I changed it to `__str__` as you suggested and added angular brackets to the string returned by `__repr__` following the official python guideline: https://docs.python.org/3/library/functions.html#repr